src/modules/Providers/ProvidersPage.tsx

```diff
@@ -151,10 +151,10 @@ const Page: React.FC<{
   const showEmptyState = (dataSource[0]?.length ?? 0) === 0;
 
   return showEmptyState ? (
-    <EmptyStateProviders />
+    <EmptyStateProviders namespace={namespace} />
```

Comment:
what happen when namespace is "all namespace"

Reply:
in 'All namespaces'  we have `namespace === undefined` (a legal value passed from the page props)